DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Raw/LevelConversionToSpeckle.cs

```diff
@@ -5,31 +5,23 @@ namespace Speckle.Converters.RevitShared.ToSpeckle;
 
 public class LevelConversionToSpeckle : IRawConversion<DB.Level, SOBR.RevitLevel>
 {
-  private readonly CachingService _cachingService;
   private readonly ToSpeckleScalingService _scalingService;
 
-  public LevelConversionToSpeckle(CachingService cachingService, ToSpeckleScalingService scalingService)
+  public LevelConversionToSpeckle(ToSpeckleScalingService scalingService)
```

Comment:
Maybe we should invert this? ScalingServiceToSpeckle @AlanRynne ?

Reply:
I would be inclined to call it ScalingServiceToSpeckle as it sounds more like proper english 😄

Reply:
okay, renamed